VK/VK/Model/Model.swift

```diff
@@ -0,0 +1,28 @@
+// Model.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+/// Описание пользователя.
+struct User: NetworkUnit {
+    var name: String
+    var description: String
+    var avatarImageName: String
+    var unitImageNames: [String]?
```

Comment:
```suggestion
    var unitImageNames: String
```
Массив в моделе лучше не использовать, поправь это в протоколе и в остальных структурах